renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -3205,7 +3205,20 @@ ShaderDebugTrace GLReplay::DebugThread(uint32_t eventID, uint32_t groupid[3], ui
 
 const GLHookSet &GetRealGLFunctions();
 
+#if defined(RENDERDOC_SUPPORT_GL)
```

Comment:
No problem. I've merged this locally and made a follow up commit, I'll push shortly and close this PR.